.babelrc

```diff
@@ -28,6 +27,88 @@
       "include": [
         "NODE_ENV"
       ]
+    }],
+    ["babel-plugin-transform-builtin-classes", {
```

Comment:
This awesome plugin is key to make it work
https://github.com/WebReflection/babel-plugin-transform-builtin-classes